etc/vlc.profile

```diff
@@ -25,5 +25,6 @@ private-bin vlc,cvlc,nvlc,rvlc,qvlc,svlc
 private-dev
 private-tmp
 
+# memory-deny-write-execute
```

Comment:
Could you add a note like:
`# mdwe is disabled due to breaking hardware accelerated decoding`
and if you want also add noexec to cvlc.profile